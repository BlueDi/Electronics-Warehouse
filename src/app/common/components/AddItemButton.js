import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import { Button, Icon } from 'semantic-ui-react';

class AddItemButton extends Component {
  render() {
    return (
      <Link to="/addNewItem">
        <Button
          icon
          labelPosition="left"
          style={{ backgroundColor: '#87DC8E' }}
        >
          <Icon name="plus" />
          Add an item
        </Button>
      </Link>
    );
  }
}

export default AddItemButton;
