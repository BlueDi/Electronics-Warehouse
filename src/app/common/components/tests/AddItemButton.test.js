import React from 'react';
import renderer from 'react-test-renderer';
import AddItemButton from '../AddItemButton';

test('AddItemButton Button Render', () => {
  const component = renderer.create(<AddItemButton />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
