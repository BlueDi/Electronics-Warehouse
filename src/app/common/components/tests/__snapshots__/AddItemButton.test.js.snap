// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItemButton Button Render 1`] = `null`;
